src/main/java/com/example/onlinestore/user/UserRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.example.onlinestore.user;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, Long> {    
```

Comment:
Repositories should not be `public`:
```suggestion
interface UserRepository extends JpaRepository<User, Long> {    
```

Reply:
what if another package requires access?